assets/timeTable.js

```diff
@@ -186,8 +192,15 @@ jQuery(document).ready(function ($) {
       this.openEditTimeLogModal();
 
       // Set date today
-      let today = new Date().toISOString().split("T")[0];
-      this.modalInputDate.val(today);
+      let currentWeekNumber = new Date().getWeek();
+      let viewWeekNumber = parseInt(this.viewCurrentWeek, 10);
```

Comment:
what is "viewcurrentweek"?

Reply:
I see that the title might be a bit misleading, but it is the week number of the week in view.

Reply:
Fixed in other branch